Sources/SwerverCore/Server/SwerverCore.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Socket
+import SwiftyBeaver
+
+let logger = SwiftyBeaver.self
+
+public class Swerver {
+    
+    public static let bufferSize = Socket.SOCKET_DEFAULT_READ_BUFFER_SIZE
+    
+    private let port: Int
+    private var listenSocket: Socket!
+    private var continueListening = true
+    
+    private let clientRequestHandler = ClientRequestHandler()
+    private let console = ConsoleDestination()
```

Comment:
What is a `ConsoleDestination`?

Reply:
that's part of the SwiftyBeaver, a logger I'm using throughout development

Reply:
:+1: on introducing a logger so early though. Definitely make sure that no log statements are being outputted when running the tests.